ada_feeding/ada_feeding/helpers.py

```diff
@@ -35,6 +54,42 @@
 MOVE_TO_NAMESPACE_PREFIX = "move_to"
 
 
+def quat_between_vectors(vec_from: Vector3, vec_to: Vector3) -> Quaternion:
```

Comment:
Trusting that this is correct, I didn't check the math on this. In the future we can consider writing a unit test for this to check it.

Reply:
Added issue #110